SPECS-EXTENDED/rubygem-puppet-resource_api/rubygem-puppet-resource_api.spec

```diff
@@ -0,0 +1,36 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
+%global gem_name puppet-resource_api
+
+Summary:        Provides a simple way to write new native resources for puppet
+Name:           rubygem-%{gem_name}
+Version:        1.8.14
+Release:        1%{?dist}
+Group:          Development/Languages
+License:        MIT
+Vendor:		Microsoft Corporation
+Distribution:	Mariner
+URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
+Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+BuildRequires:  ruby
+
+%description
+This library provides a simple way to write new native resources for puppet.
+
+%prep
+%setup -q -c -T
+
+%build
+
+%install
+gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{SOURCE0}
+
+%files
+%defattr(-,root,root,-)
+%{gemdir}
```

Comment:
Is there any license file to add with the `%license` macro? Something similar to what Henry did in `rubygem-multi_json.spec`.

Reply:
added